deploy/olm-catalog/argocd-operator/0.7.0/argocd-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -3,14 +3,14 @@ kind: Service
 metadata:
   creationTimestamp: null
   labels:
-    control-plane: controller-manager
+    control-plane: argocd-operator
```

Comment:
I think this should stay controller-manager

Reply:
done, thanks